geomesa-accumulo/geomesa-accumulo-tools/src/main/scala/org/locationtech/geomesa/accumulo/tools/AccumuloRunner.scala

```diff
@@ -93,24 +98,42 @@ object AccumuloRunner extends Runner {
       if (p.zookeepers == null) {
         p.zookeepers = zookeepers
       }
-      if (p.instance == null) {
-        p.instance = try {
-          // This block checks for the same system property which Accumulo uses for Zookeeper timeouts.
-          //  If it is set, we use it.  Otherwise, a timeout of 5 seconds is used.
-          //  Don't give default to GeoMesaSystemProperty so .option will throw a None
-          val lookupTime: Long = SystemProperty("instance.zookeeper.timeout").option.flatMap{ p =>
-            Try { java.lang.Long.parseLong(p) }.toOption
-          }.getOrElse(5000L)
+      try {
+        // This block checks for the same system property which Accumulo uses for Zookeeper timeouts.
+        //  If it is set, we use it.  Otherwise, a timeout of 5 seconds is used.
+        //  Don't give default to GeoMesaSystemProperty so .option will throw a None
+        val lookupTime: Long = SystemProperty("instance.zookeeper.timeout").option.flatMap{ p =>
+          Try { java.lang.Long.parseLong(p) }.toOption
+        }.getOrElse(5000L)
 
-          Command.user.debug(s"Looking up Accumulo Instance Id in Zookeeper for $lookupTime milliseconds.")
-          Command.user.debug("You can specify the Instance Id via the command line or\n" +
-            "change the Zookeeper timeout by setting the system property 'instance.zookeeper.timeout'.")
+        Command.user.debug(s"Looking up Accumulo Instance Id in Zookeeper for $lookupTime milliseconds.")
+        Command.user.debug("You can specify the Instance Id via the command line or\n" +
+          "change the Zookeeper timeout by setting the system property 'instance.zookeeper.timeout'.")
 
-          import scala.concurrent.duration._
-          Await.result(Future(HdfsZooInstance.getInstance().getInstanceName)(ExecutionContext.global),  lookupTime.millis)
+        import scala.concurrent.duration._
+        lazy val ff: Either[Throwable, String] = try {
```

Comment:
Unfortunately that doesn't work. The way I have it is the only way I could find that would both catch the exception in the Future and when it comes back. Originally it was throwing the stackstrace twice. Using a Try with .get here only catches the stackstace after the future returns.